### ISSUE TITLE
Change color of background of deleted posts

### ISSUE THREAD

**Issue by MasterOdin** [Submitty/Submitty#3628]
> @bmcutler This looks okay but if you don't mind can you please let me change background to little > bit of light color as test is not readable as color contrast ratio `3.77:1`. If possible make background `C9C9C9`.
> @pbailie what are your views?

_Originally posted by @drumilpatel2000 in https://github.com/Submitty/Submitty/pull/3612#issuecomment-486901993_

Created this issue to discuss if we want to change the color to improve the contrast or some potential other change. We should come to a set conclusion on what we want to do before any PR gets made.

**Comment by pbailie** [comment]
Contrast ratio is important for the visually impaired, which includes color blindness.  I work towards a common goal of 4.5:1 as that will cover a vast majority of cases for Submitty.

Cases are actually based on not only background and text color, but also text size.  But as the vast majority of text in submitty is 16px, I believe 4.5:1 contrast ratio is a simple but effective standard to work with.

If anyone ever wants to test their color scheme for contrast ratio — this is a good site:  https://webaim.org/resources/contrastchecker/

Just bear in mind that this only tests red, green, blue, but not alpha.  (e.g. 24 bit color codes, not 32 bit).

**Comment by drumilpatel2000** [comment]
@pbailie Do you agree of changing background color to `C9C9C9` ?This will make color contrast `5.1.8:1`.
